flang/lib/Lower/OpenMP.cpp

```diff
@@ -590,6 +590,8 @@ class ClauseProcessor {
   bool processSectionsReduction(mlir::Location currentLocation) const;
   bool processTo(llvm::SmallVectorImpl<DeclareTargetCapturePair> &result) const;
   bool
+  processEnter(llvm::SmallVectorImpl<DeclareTargetCapturePair> &result) const;
```

Comment:
Nit: It would be good to keep the alphabetic order of declarations here and definitions below, so I'd add this between `processDeviceType` and `processFinal`.

Reply:
Thanks for the review @skatrak. I moved it between `processDepend` and `processIf` because `enter` is a repeatable clause.